### DISCUSSION TITLE
A way to disable these "recent pushes" notifications to "Compare & pull request"

### DISCUSSION THREAD

**Discussion by karl-police** [community/community#34225] [Repositories]
I created a repository, that acts as an archive to view a specific previous version. And I get these notifications on the screen when I go to the repository, and it's annoying then.

![image](https://user-images.githubusercontent.com/12023782/192315384-9fe34d82-72e4-4f4d-835b-68f17c28d198.png)

Would be cool if this notification wouldn't always appear for every repository, or not appear at all.

**Comment by ElectriPixie** [comment]
I'd rather not have them at all if they can't be per branch or configured in some less insane way

**Comment by krischik** [comment]
Same here. At least there should be an "ignore this" button but having an ugly yellow box on the screen when you don't actually want to make a pull request is an eye sore,

**Comment by pphhtt8887777-ctrl** [comment]
> I created a repository, that acts as an archive to view a specific previous version. And I get these notifications on the screen when I go to the repository, and it's annoying then.
> 
> ![image](https://user-images.githubusercontent.com/12023782/192315384-9fe34d82-72e4-4f4d-835b-68f17c28d198.png)
> 
> Would be cool if this notification wouldn't always appear for every repository, or not appear at all.

>✓

**Comment by pphhtt8887777-ctrl** [comment]
>[✓]
